frontend/mirage/factories/timed-subscription-order.js

```diff
@@ -7,7 +7,7 @@ export default Factory.extend({
   duration() {
     return DjangoDurationTransform.create().serialize(
       moment.duration({
-        hours: faker.random.number(100),
+        hours: faker.random.number(200),
```

Comment:
this is actually useless. you can remove this since no time on the project is calculated with orders.

Reply:
at least not on the mock server.